packages/@uppy/core/src/UIPlugin.ts

```diff
@@ -176,8 +179,12 @@ class UIPlugin<
    * so this.el and this.parent might not be available in `install`.
    * This is the case with @uppy/react plugins, for example.
    */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  render(state: Record<string, unknown>): ComponentChild {
+  render(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    state: Record<string, unknown>,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    container: HTMLElement,
```

Comment:
should we use object argument instead, which is more flexible to extend in the future if needed?

```suggestion
{ state, container } : {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    state: Record<string, unknown>,
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    container: HTMLElement,
    }
```